lizmap/modules/lizmap/classes/lizmapFtsSearch.listener.php

```diff
@@ -38,13 +38,24 @@ protected function checkLizmapFts()
             $profile = null;
         }
 
+        // Lizmap search available only for PostgreSQL
+        $cnx = jDb::getConnection($profile);
+        $dbms = $cnx->dbms;
```

Comment:
why creating a variable, since it is used only once?

Reply:
it is indeed useless. I will remove this unnecessary step